### PR TITLE
fixing log messages

### PR DESCRIPTION
**Pull request by cpanato**
#### Summary
fixing log messages

#### Ticket Link
n/a

### CODE REVIEW

**Comment by cpanato** [comment]
@jasonblais asked in the dev-priv channel and @hmhealey said we are not translating log messages anymore, so don't need to add those in the en.json file

**Comment by jasonblais** [comment]
Cool - is it okay for me to make a few word tweaks on the strings?

**Comment by cpanato** [comment]
yes sir @jasonblais

File: app/notification.go [hunk 1]
```diff
@@ -284,7 +284,7 @@ func (a *App) SendNotifications(post *model.Post, team *model.Team, channel *mod
 	if *a.Config().EmailSettings.SendPushNotifications {
 		pushServer := *a.Config().EmailSettings.PushNotificationServer
 		if license := a.License(); pushServer == model.MHPNS && (license == nil || !*license.Features.MHPNS) {
-			mlog.Warn("api.post.send_notifications_and_forget.push_notification.mhpnsWarn FIXME: NOT FOUND IN TRANSLATIONS FILE")
+			mlog.Warn("Failed to get the license or check the MHPNS feature. Push notification is disabled")
```
Comment (jasonblais) [inline app/notification.go]: Is it that Mattermost cannot
 - get the EE license
 - confirm if the Mattermost Hosted Push Notification Service is enabled

and hence push notifications have been disabled?

File: app/notification.go [hunk 2]
```diff
@@ -365,7 +365,7 @@ func (a *App) SendNotifications(post *model.Post, team *model.Team, channel *mod
 
 		var infos []*model.FileInfo
 		if result := <-fchan; result.Err != nil {
-			mlog.Warn(fmt.Sprint("api.post.send_notifications.files.error FIXME: NOT FOUND IN TRANSLATIONS FILE", post.Id, result.Err), mlog.String("post_id", post.Id))
+			mlog.Warn(fmt.Sprint("Faile to get the fileInfo", post.Id, result.Err), mlog.String("post_id", post.Id))
```
Comment (jasonblais) [inline app/notification.go]: Is this for files uploaded to Mattermost

File: app/import_functions.go [hunk 3]
```diff
@@ -491,10 +491,10 @@ func (a *App) ImportUser(data *UserImportData, dryRun bool) *model.AppError {
 	if data.ProfileImage != nil {
 		file, err := os.Open(*data.ProfileImage)
 		if err != nil {
-			mlog.Error(fmt.Sprint("api.import.import_user.profile_image.error FIXME: NOT FOUND IN TRANSLATIONS FILE", err))
+			mlog.Error(fmt.Sprint("Unable to open the profile image.", err))
```
Comment (jasonblais) [inline app/import_functions.go]: Updated these to `Unable to ... ` - let me know if any concerns.

File: app/notification_email.go [hunk 4]
```diff
@@ -89,7 +89,7 @@ func (a *App) sendNotificationEmail(post *model.Post, user *model.User, channel
 
 	a.Go(func() {
 		if err := a.SendMail(user.Email, html.UnescapeString(subjectText), bodyText); err != nil {
-			mlog.Error(fmt.Sprint("api.post.send_notifications_and_forget.send.error FIXME: NOT FOUND IN TRANSLATIONS FILE", user.Email, err))
+			mlog.Error(fmt.Sprint("Error to send the email", user.Email, err))
```
Comment (jasonblais) [inline app/notification_email.go]: This is an email sent for mentions?

File: cmd/mattermost/commands/server.go [hunk 5]
```diff
@@ -243,7 +243,7 @@ func runSessionCleanupJob(a *app.App) {
 
 func resetStatuses(a *app.App) {
 	if result := <-a.Srv.Store.Status().ResetAll(); result.Err != nil {
-		mlog.Error(fmt.Sprint("mattermost.reset_status.error FIXME: NOT FOUND IN TRANSLATIONS FILE", result.Err.Error()))
+		mlog.Error(fmt.Sprint("Error to reset the status", result.Err.Error()))
```
Comment (jasonblais) [inline cmd/mattermost/commands/server.go]: Is this to reset the server status?

File: app/notification.go (hunk 1 above)
Comment (cpanato) [reply app/notification.go]: MM cannot get the license or the license does not have the NHPNS feature

File: app/notification.go (hunk 2 above)
Comment (cpanato) [reply app/notification.go]: to add the images info in the push notification

File: app/import_functions.go (hunk 3 above)
Comment (cpanato) [reply app/import_functions.go]: sounds good

File: app/notification_email.go (hunk 4 above)
Comment (cpanato) [reply app/notification_email.go]: yes

File: cmd/mattermost/commands/server.go (hunk 5 above)
Comment (cpanato) [reply cmd/mattermost/commands/server.go]: yes

File: app/import_functions.go (hunk 3 above)
Comment (hmhealey) [inline app/import_functions.go]: Should these be putting the error in a separate argument? Like
```go
mlog.Error("Unable to open the profile image.", mlog.Any("err", err))
```

File: app/notification.go (hunk 1 above)
Comment (jasonblais) [reply app/notification.go]: By `doesn't have`, is that possible? Just wondering why an admin might hit that error, or if it's someone who would have forked Mattermost and removed the feature?

**Comment by cpanato** [comment]
@jasonblais PTAL

File: app/notification.go (hunk 1 above)
Comment (jasonblais) [reply app/notification.go]: @cpanato Figuring out what would be the appropriate text. Push notifications can be enabled without an EE license, and I'm not sure why someone wouldn't have the HPNS service available..

Perhaps

```
Push notifications are disabled. Go to System Console > Notifications > Mobile Push to enable them.
```

?

**Comment by jasonblais** [comment]
Added a question above ^ . https://github.com/mattermost/mattermost-server/pull/9341#discussion_r217688984

File: app/notification.go (hunk 1 above)
Comment (cpanato) [reply app/notification.go]: sounds good this message

**Comment by jasonblais** [comment]
Ready to merge @cpanato

**Pull request merged**